packages/office-addin-dev-settings/README.md

```diff
@@ -3,17 +3,20 @@
 Provides the ability to configure developer settings for Office Add-ins.
 
 ## Command-Line Interface
-* [appcontainer](#appcontainer)
-* [clear](#clear)
-* [debugging](#debugging)
-* [live-reload](#live-reload)
-* [register](#register)
-* [registered](#registered)
-* [runtime-log](#runtime-log)
-* [sideload](#sideload)
-* [source-bundle-url](#source-bundle-url)
-* [unregister](#unregister)
-* [webview](#webview)
+- [Office-Addin-Dev-Settings](#office-addin-dev-settings)
```

Comment:
Why are we changing the formatting here?

Reply:
This is weird, I didn't intend to change this. When I tried saving this portion of the readme back to it's original form, it automatically changed it to this format

Reply:
![image](https://user-images.githubusercontent.com/3607629/104350070-95ddf880-54b8-11eb-9886-5afc98b89627.png)

Reply:
Upon saving:

![image](https://user-images.githubusercontent.com/3607629/104350123-a9895f00-54b8-11eb-8730-77dadc65db08.png)

Reply:
Is this change intended? I don't see a need to make changes here.

Reply:
Can you just revert that change to the formatting and recommit?

Reply:
Reverted these changes, but had to use notepad to make them. Weird, I know.

Reply:
You should be able to see these diffs locally even before you push with `git status` and `git difftool`. I like to use `git difftool` to review changes before committing or pushing, as I can make changes and catch stuff. Also, if you edit a file while viewing the diffs, the edits will be applied to the unstaged files, so you can undo specific things easily this way. Glad to show an example some time.